rcgen/examples/simple.rs

```diff
@@ -0,0 +1,38 @@
+#![allow(clippy::complexity, clippy::style, clippy::pedantic)]
```

Comment:
I think this could be removed. I only see a handful of clippy findings in this code and they look easy to fix.